dependabot_file.py

```diff
@@ -61,7 +62,17 @@ def build_dependabot_file(
 
     if existing_config:
         dependabot_file = existing_config.decoded.decode("utf-8")
+        directory_line = next(
+            line for line in dependabot_file.splitlines() if "directory:" in line
+        )
+        indent = " " * (len(directory_line) - len(directory_line.lstrip()))
+        if len(indent) < 2:
+            print(
+                "Invalid dependabot.yml file. No indentation found. Skipping {repo.full_name}"
+            )
+            return None
     else:
+        indent = " " * 2
```

Comment:
could we move the `2` to the top of the file and call it something like `DEFAULT_INDENT = 2`? wdyt?